source/module_base/tool_quit.cpp

```diff
@@ -47,7 +47,7 @@ void QUIT(void)
 
     ModuleBase::Global_File::close_all_log(GlobalV::MY_RANK);
 
-    if (GlobalV::MY_RANK==0)
+    //if (GlobalV::MY_RANK==0)
     {
         ModuleBase::Memory::print_all( GlobalV::ofs_running ) ;
```

Comment:
I suggest not to print memory in program abort, because program will give clear advise to users about the problem, memory should be printed only when program ends normally. 
@mohanchen what is your suggestion for this PR?

Reply:
I agree

Reply:
@dyzheng. I removed the code for printing memory info in the function of QUIT.